README.md

```diff
@@ -102,7 +102,7 @@ En ese PR deben incluirse los siguientes cambios:
 2. Modificar este README.md para que los ejemplos usen la nueva versión `X.Y.Z`.
 3. Actualizar la versión declarada en `package.json` y `src/onepay-lib-version.js` para que tenga el valor `X.Y.Z` y ejecutar `npm run build`.
```

Comment:
Don't forget to remove this line in the README (given that you automated all of this)

Reply:
Done with b207176